_posts/2015-07-18-open-code-of-conduct.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Announcing the Open Code of Conduct
+author: bkeepers
+---
+
+We strongly believe open source communities should be a welcoming place for all participants. Through our experiences within the TODO Group, we strongly believe that a code of conduct helps set the ground rules for participation in communities and helps build a culture of respect. By adopting and honoring a code of conduct, communities can communicate their values, set expectations and outline a process for dealing with unwelcome behavior when it arises.
+
+Based on our experiences, we are proud to share the [Open Code of Conduct](http://todogroup.org/opencodeofconduct), an easy-to-reuse code of conduct template for open source communities. Some members of the TODO Group have already [adopted this code of conduct](https://github.com/todogroup/opencodeofconduct#what-companies-support-or-use-the-open-code-of-conducts) for the projects they maintain and more plan to do so in the near future.
```

Comment:
The first paragraph uses "…through our experiences…", and this starts "Based on our experiences…". Does that need repeated?